gf-webapp-admin/src/component/rooms/roomSimplified/RoomSimplified.module.scss

```diff
@@ -1,18 +1,18 @@
-@import "../../../Index.scss";
+@import "src/Index.scss";
 
 .roomSimplified {
   cursor: pointer;
   overflow: hidden;
   box-shadow: 0px 0px 10px 5px rgba(34, 60, 80, 0.2);
+  height: 100%;
+  width: 33%;
+  min-height: 100px;
 }
 
 .imageContainer {
   position: relative;
   display: flex;
   flex-wrap: wrap;
-  height: 100%;
```

Comment:
Why are you changing styles? Grid looks better than flex with adaptive. Please back old styles

Reply:
The main reason why I changed grid to flex was that blocks with room rendering with different size depending on the number of rooms. I think it's a bad idea to back old styles because it's wrong view.